admin/includes/css/developers_tool_kit.css

```diff
@@ -14,3 +14,7 @@
 .dtk-contextline {font-family: Verdana, sans-serif;}
 .dtk-linenum {font-family: Verdana, sans-serif;}
 .dataTableGroupChange {border-top: 2px solid #000 !important;}
+table.results {border:2px solid;}
+table.blank {width:100%;}
```

Comment:
There's no table here with a `blank` class, please remove.

If it turns out to be needed, suggest using `w-100` as the class name, since that's a Bootstrap 4/5 included class for a 100% width.

Reply:
done